src/main/io/gps.c

```diff
@@ -245,11 +255,37 @@ typedef struct ubxCfgNav5_s {
     uint8_t reserved1[5];
 } ubxCfgNav5_t;
```

Comment:
```suggestion
} ubxCfgNav5m8_t;
```

Reply:
What changes across protocol versions is that additional fields are added to the areas originally defined as reserved bytes. Fields are only added, not changed. The M9 & M10 5.00 protocol specs are better at annotating at which protocol version each field was added.

Protocol version 16.00 was the last to add a new field (utcStandard). However the M8 platform includes protocols from version 15.00 through 23.01. So labeling the message struct as 'M8' isn't really accurate, it would have to be labeled with the protocol version to distinguish it.

Fortunately, the overall size of the NAV5 message remains the same across protocol versions, and newer formats remain backwards compatible, so the UBX protocol doesn't define versions for NAV5.

If we define our message struct to be the latest definition (as of protocol 16.00+), we would only need one message struct definition. The code will still need to know whether it's working with a module that supports any of the new fields it might be trying to use, but that would also be true if we had multiple message struct definitions.

This way we wouldn't need to complicate our structure definitions and can just have one NAV5 that isn't qualified by module or protocol version.

Reply:
Resolved, I think.